addons/base_onboarding/models/base_onboarding.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from .base_onboarding_state import ONBOARDING_STATES
+from odoo.tools.json import scriptsafe as json_safe
+
+
+class Onboarding(models.Model):
+    _name = 'base.onboarding'
+    _description = 'Onboarding'
+
+    name = fields.Char('Name of the onboarding')
+    route_name = fields.Char(
+        'Short name to use in route', default=lambda s: s.name.replace(' ', '') if s.name else 'route')
+    step_ids = fields.One2many('base.onboarding.step', 'onboarding_id', 'Onboarding steps')
+
+    panel_background_color = fields.Selection(
+        [('orange', 'Orange'), ('blue', 'Blue')], default='orange')
+    panel_background_image_url = fields.Char("URL for the panel's background image")
+    panel_properties = fields.Text(
+        'Panel template properties', compute='_compute_panel_properties', store=False, default='')
+    panel_close_action_name = fields.Char('Action to execute when closing the panel')
+
+    is_per_company = fields.Boolean(
+        'Should any of the onboarding steps be done for each company', compute='_compute_is_per_company',
+        readonly=True)
+
+    is_done = fields.Boolean('Is completed', compute='_compute_is_done')
+    is_closed = fields.Boolean('Is completed', compute='_compute_is_closed')
+
+    onboarding_state = fields.Selection(
```

Comment:
I would probably store it, to speedup computation / access. (like "have I some onboarding to do", should not require to fetch all steps each time).